CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Change Log
 
 ## [master](https://github.com/arangodb/go-driver/tree/master) (N/A)
+- Add support for streaming the response body by the caller.
```

Comment:
Please, add information that it is V2 change

Reply:
Also, adjust PR name to [Feature] [V2] ...

Reply:
changed